ocaml/xapi-consts/api_errors.ml

```diff
@@ -1290,6 +1290,9 @@ let invalid_update_sync_day = "INVALID_UPDATE_SYNC_DAY"
 
 let no_repositories_configured = "NO_REPOSITORIES_CONFIGURED"
 
+let host_pending_mandatory_guidances_not_empty =
+  "HOST_PENDING_MANDATORY_GUIDANCE_NOT_EMPTY"
```

Comment:
How about "NO_HOST_PENDING_MANDATORY_GUIDANCE_ALLOWED"?

Reply:
I feel `host_pending_mandatory_guidances_not_empty` is clearer for the state of the problem.
While for the pattern: `NO_xxx_ALLOWED`, it gives me a wrong feeling that, something is being perfomed/generated, and it is not allowed, for example, as the errors we already have:
` 195 let operation_not_allowed = "OPERATION_NOT_ALLOWED"`
`1125 let no_more_redo_logs_allowed = "NO_MORE_REDO_LOGS_ALLOWED"`